openconnect_sso/cli.py

```diff
@@ -60,6 +60,12 @@ def create_argparser():
         default="",
     )
 
+    auth_settings.add_argument(
```

Comment:
This setting is not related to authentication, please use the global parser object for that.

Reply:
Ack. Will do.